apps/website/docs/pipelines/release-azure-webapp.md

```diff
@@ -0,0 +1,73 @@
+---
+sidebar_position: 50
+---
+
+# Deploying a Node.js Application to Azure
```

Comment:
question: is this the same page that will document container app deploy?

Reply:
no, the container app deploy right now has a different workflow (`release-azure-containerapp-v1.yaml`)

Reply:
then it's maybe advisable to have a more specific title ie. "Deploying a Node.js Application to Azure App Service or Function" ?

Reply:
I still see the "old" title